dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/ISimplifiedRowStachExtension.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+
+namespace FactSet.Protobuf.Stach.Extensions
+{
+    public interface ISimplifiedRowStachExtension : IStachExtension
+    {
+        List<dynamic> ConvertToDynamicObject();
```

Comment:
Is it better to call this method `ConvertToDynamicObjects`? Also, can you add some documentation for this method?